services/core-api/app/api/services/email_service.py

```diff
@@ -142,7 +152,9 @@ def send_email(cls,
         EmailService.perform_health_check()
 
         # If the sender is the MDS no-reply email address, add the MDS no-reply signature to the email body.
-        if sender == Config.MDS_NO_REPLY_EMAIL:
+        if proponent:
+            body += MINESPACE_NO_REPLY_SIGNATURE
+        elif sender == Config.MDS_NO_REPLY_EMAIL:
             body += MDS_NO_REPLY_SIGNATURE
```

Comment:
hmmmm, I'm not sure about this logic. The trigger for all emails at this point comes from the proponent.

Reply:
Ohhh, nvm, I see what you did. 'Proponent' will default to false. Okay, this works! 

Can we update the param name as it's a little confusing since the proponent is the trigger for all emails. Maybe 'send_minespace_email' or 'minespace_email', or 'send_to_proponent'... something like that?